### DISCUSSION TITLE
Split fan header won't hold RPM

### DISCUSSION THREAD

**Discussion by DoctorBatman31** [Rem0o/FanControl.Releases#1337] [Q&A] [answered]
Hi all,

Hoping I can get some tips or suggestions for a problem I'm having.

I am having an issue where Fan Control for one of my chassis fan headers is not holding a consistent rpm. The fan will run at the correct rpm for a little bit, drop to zero rpm, and then ramp back up before repeating the sequence. I have a second fan header matched with the same curve and it holds the rpm consistently, so I know its not an issue with the curve itself.

Now, the issue I presume is regarding the fact that the problem fans are hooked up to the header with a splitter. (x2 120mm PWM fans to x1 PWM header) When I disconnect one of the fans from the splitter, so that only one of them is running, the problem goes away. Also, I know that the problem is not that the fans are drawing too much power from the header. (at least it shouldn't) When building the PC I checked the current draw of the fans compared to the max current from the header, and the max draw from each fan is 0.14A, and the max draw allowed from the header is 1.0A. (If some one would like to do a sanity check on this, see the component list below.) I've been running this orientation for a bit using the bios fan control, and didn't have any issues. I've also tried changing the fan mode in the bios between Auto/DC/PWM, and saw no change. I also set the bios fan curve to "standard".

I would like to avoid getting a fan controller, as the system is sff, and I don't really want to introduce any more components or wires.
Any suggestions or troubleshooting tips would be greatly appreciated.

Components:
Noctua NF-A12x25 (x2)
Asus ROG STRIX B660-I GAMING WIFI

**Comment by Rem0o** [comment, marked as answer]
Can you send your config? What's the start % and stop % of these fans? What % are you holding them at?

**Reply by DoctorBatman31** [reply]
I've tried a couple different curves, but the attached config has two which I have been mostly testing with. I've been playing around with increasing and decreasing the temps, %, hysteresis, and response, but keep seeing the same pattern in the fan operation (both with the graph and linear curves).

Fan Start/Stop %:
- The start % from the autodetect showed 18%
- The stop % from the autodetect showed 9%

Config:
[Custom Curve - Testing.zip](https://github.com/Rem0o/FanControl.Releases/files/10147443/Custom.Curve.-.Testing.zip)

**Reply by Rem0o** [reply]
The fact one of the fans stops and not the other on the same header throws me off, as they can't be controlled separately, it's both or nothing.  Do you have a spare header? If so, I would recommend running 1 fan per channel. I know that goes around your problem, but I can't think of a reason why it would act like that.

**Reply by DoctorBatman31** [reply]
Both of the fans connected via the splitter stop and start simultaneously.

In the config file "Case Fan - Top (x2)" refers to the two fans connected via the splitter to fan header #1. "Case Fan - Bottom" is a single fan connected directly to its own header (fan header #2). The bottom fan runs as intended. But the top two both start and stop simultaneously, when they should be running consistently.

Sorry for the confusion, hope this clears it up a bit.

**Reply by Rem0o** [reply]
^ Thanks for the clarification. Still, if the problem goes away when you only have a single fan plugged in, something is off with that splitter.  Is the RPM reporting ok, like does it report only a single of the two headers?

In any case, if you got a spare header, try 1 fan per header.

**Reply by DoctorBatman31** [reply]
Unfortunately I don't have another header. Only the CPU and the two chassis headers. I was starting to think the same thing though. I'm going to order a new splitter, so we'll see if that makes any difference.

I'll update if anything changes.

**Reply by Rem0o** [reply]
By any chance, try and remove the paired RPM sensor on that control, and see if it changes anything.

**Reply by DoctorBatman31** [reply]
You're my savior.

After removing the paired RPM sensor it works like a charm.

Thanks for the help, really appreciate it.

**Reply by Rem0o** [reply]
@DoctorBatman31  Ah, my instinct is still sharp ;)

Fan Control uses the matched RPM sensor as a reference if something is off.  If it detects for example the RPM is at 0, it will react to it and try to restart the fan and so on. 

My guess is that your splitter messes with the RPM, and sometimes it reports something funny, which fan control picks up.
